include/boost/geometry/strategies/relate/services.hpp

```diff
@@ -52,6 +52,30 @@ struct strategy_converter
 }} // namespace strategies::relate
 
 
+namespace strategies { namespace set_operations
+{
+namespace services
+{
+
+template
+<
+    typename Geometry1,
+    typename Geometry2,
+    typename CSTag1 = typename geometry::cs_tag<Geometry1>::type,
+    typename CSTag2 = typename geometry::cs_tag<Geometry2>::type
+>
+struct default_strategy
```

Comment:
I haven't chacked this out but I think it is not required to define this struct and its specializations. AFAIU the only thing you need is template alias for `strategies::relate::services::default_strategy`:
```
template
<
    typename Geometry1,
    typename Geometry2,
    typename CSTag1 = typename geometry::cs_tag<Geometry1>::type,
    typename CSTag2 = typename geometry::cs_tag<Geometry2>::type
>
using default_strategy = strategies::relate::default_strategy<Geometry1, Geometry2, CSTag1, CSTag2>;
```

or rather four aliases, one for each set operation (see other comment).